README.md

```diff
@@ -152,11 +153,15 @@ feel free to populate the rest of the manifest so that your package works proper
 manager. Just note that the Swift Gazelle plugin does not use the manifest to generate Bazel build
 files, at this time.
 
-### 4. Run `bazel mod tidy`.
+### 4. Run `swift package resolve`
```

Comment:
I think that we want to recommend `swift package update`. The `resolve` command will always try to find the latest version of the transitive deps. That is what the `swift_update_packages` macro would do by default.

Reply:
Done.